spec.md

```diff
@@ -1436,11 +1587,9 @@ wire w: UInt
 invalidate w
 ```
 
-Invalidate statements can be applied to any circuit component of any
-type. However, if the circuit component cannot be connected to, then the
```

Comment:
As far as the rest of the spec indicates, no circuit component cannot be connected to.

Arguably submodule instances or memories were initially considered, but it is not explicitly forbidden. (Or if it is, we need to reference where and why).

Reply:
>  no circuit component cannot be connected to.

That is not true, you are allowed to not connect to e.g. the `input` of your top level module in the circuit.

Reply:
Ah. Thank you. I was trying to work out when this would be restricted.

Is there any problem with me changing this to a component that has Sink or Duplex flow? "Cannot be connected to" puts too much obligation on the reader to read between the lines.